uk.co.compucorp.civicrm.hrleaveandabsences/views/shared/directives/leave-request-popup.html

```diff
@@ -16,181 +16,202 @@ <h4 class="modal-title">
                     <div class="col-xs-4">
-                      <i
-                        class="fa chr_leave-request-modal__chevron"
-                        aria-hidden="true"
-                        ng-class="{ 'fa-chevron-right': !$ctrl.uiOptions.isChangeExpanded, ' fa-chevron-down': $ctrl.uiOptions.isChangeExpanded }"></i>
                       Change
                     </div>
                     <div class="col-xs-8">{{$ctrl.balance.change.amount}}</div>
                   </div>
-                  <table
-                    class="table chr_leave-request-modal__table"
-                    ng-show="$ctrl.uiOptions.isChangeExpanded">
-                    <thead>
-                    <tr>
-                      <th>Date</th>
-                      <th>Selection</th>
-                      <th>Amount</th>
-                    </tr>
-                    </thead>
-                    <tbody>
-                    <tr ng-repeat="breakdown in $ctrl.pagination.filteredbreakdown">
-                      <td>{{breakdown.date | date:'EEE '+ $ctrl.uiOptions.userDateFormat}}</td>
-                      <td>{{breakdown.type.label}}</td>
-                      <td>{{breakdown.amount}}</td>
-                    </tr>
-                    </tbody>
-                    <tfoot>
-                    <tr>
-                      <td colspan="3" class="text-center">
-                        <uib-pagination
-                          ng-if="$ctrl.pagination.totalItems"
-                          total-items="$ctrl.pagination.totalItems"
-                          ng-model="$ctrl.pagination.currentPage"
-                          class="pagination"
-                          ng-change="$ctrl.pagination.pageChanged()"
-                          direction-links="false"
-                          items-per-page="$ctrl.pagination.numPerPage"></uib-pagination>
-                      </td>
-                    </tr>
-                    </tfoot>
-                  </table>
-                </div>
-                <div class="row" ng-class="{'text-danger': $ctrl.balance.closing < 0}">
-                  <div class="col-xs-4">Closing Balance</div>
-                  <div class="col-xs-8">{{$ctrl.balance.closing}}</div>
+                  <div ng-hide="$ctrl.isLeaveType('toil')">
+                    <div
+                      class="row chr_leave-request-modal__form-group text-primary pointer"
+                      ng-click="$ctrl.uiOptions.isChangeExpanded = !$ctrl.uiOptions.isChangeExpanded">
+                      <div class="col-xs-4">
+                        <i
+                          class="fa chr_leave-request-modal__chevron"
+                          aria-hidden="true"
+                          ng-class="{ 'fa-chevron-right': !$ctrl.uiOptions.isChangeExpanded, ' fa-chevron-down': $ctrl.uiOptions.isChangeExpanded }"></i>
+                        Change
+                      </div>
```

Comment:
maybe due to code reformatter or multiple movements of absence type selection box ,